java/src/jmri/jmrit/vsdecoder/VSDecoder.java

```diff
@@ -606,6 +592,7 @@ public void disable() {
      *
      * @return boolean true if this is the default VSDecoder
      */
+    @Deprecated
```

Comment:
It's good to add a comment about the deprecation with the JMRI version where the deprecation is done. That way, we know when it was done and when we can remove the method completely.

Se the bottom of the section `Deprecating Code` on this page:
https://www.jmri.org/help/en/html/doc/Technical/RP.shtml

Reply:
Thanks, will change that.